libraries/core_libs/consensus/src/vote_manager/vote_manager.cpp

```diff
@@ -292,9 +269,6 @@ void VoteManager::verifyVotes(uint64_t pbft_round, size_t sortition_threshold, u
                               std::function<size_t(addr_t const&)> const& dpos_eligible_vote_count) {
   // Cleanup votes for previous rounds
   cleanupVotes(pbft_round);
-
-  std::vector<std::shared_ptr<Vote>> future_unverifiable_votes;
```

Comment:
The future_unverifiable_votes is for protect from malicious players. Malicious players could send huge votes in future round, like in round 1000000000000.